README.md

```diff
@@ -1,126 +1,142 @@
+  - [💻 Getting Started ](#getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [👥 Authors ](#authors)
+  - [🔭 Future Features ](#future-features)
+  - [🤝 Contributing ](#contributing)
+  - [⭐️ Show your support ](#️show-your-support)
+  - [🙏 Acknowledgments ](#acknowledgments)
+  - [📝 License ](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 School_library_Ruby <a name="about-project"></a>
+# 📖 School Library <a id="about-project"></a>
 
-**School_library_Ruby** -Imagine that you are the librarian of OOP University, and you need a tool to record what books are in the library and who borrows them
-## 🛠 Built With <a name="built-with"></a>
+This project helps to understand the basics of Ruby programming language. It is a collection of exercises and projects that will help you to understand the basics of OOPS in Ruby programming language.
 
-### Tech Stack <a name="tech-stack"></a>
+## 🛠 Built With <a id="built-with"></a>
+
+### Tech Stack <a id="tech-stack"></a>
+
+  <details>
+    <summary>Server</summary>
+    <ul>
+      <li><a href="#">Ruby</a></li>
+    </ul>
+  </details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-- Ruby
 <!-- Features -->
 
-### Key Features <a name="key-features"></a>
+### Key Features <a id="key-features"></a>
 
--  [x] Add decorator class and his method like the trim and capitalize👌
- - [x] Add Student class and his method💯
- - [x] Add Teachers class and his method👌
+- **Add Classes**
+- **Decorate a class**
+- **Set up associations**
+- **basic UI**
+- **refactor to meet SOLID principles**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
+<!-- LIVE DEMO -->
 
-## 💻 Getting Started <a name="getting-started"></a>
 
-To get a local copy up and running, follow these steps.
+<!-- GETTING STARTED -->
 
-### Prerequisites <a name="prerequisites"></a>
+## 💻 Getting Started <a id="getting-started"></a>
+### Prerequisites
 
 In order to run this project you need:
 
-```sh
-A Computer
-Code editor (VSCode,Atom etc...)
-A browser (Chrome,Mozilla,Safari etc...)
-Version Control System (Git and Github)
-NodeJS (https://nodejs.org/en/)
-```
-
-### Setup <a name="setup"></a>
+- Git and GitHub
+- A code editor
+- Ruby
 
-Clone this repository to your desired folder:
+### Setup
 
-```sh
-  cd my-folder
-  git clone https://github.com/karanJ2212/School_library_Ruby.git
-```
+To get a local copy up and running, follow these steps:
 
-### Deployment <a name="#triangular_flag_on_post-deployment"></a>
+- Create a local directory where you can clone the project
+  - `https://github.com/karanJ2212/School_library_Ruby.git`
+### Usage
 
-You can deploy this project using [Render](https://render.com/).
+To run the project, execute the following command in command line:
+> ruby main.rb
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
-
-## 👥 Authors <a name="authors"></a>
+## 👥 Author <a id="authors"></a>
```

Comment:
- [ ] [OPTIONAL]
It will be a good idea to change the heading _Author_ to _Authors_, as there are 2 contributors for this project!